srv/server.py

```diff
@@ -58,6 +58,8 @@ async def get_shapes(beg_time: int = None, end_time: int = None) -> Shapes:
                     "x": circle[2],
                     "y": circle[3],
                     "r": circle[4],
+                    "created_at": circle[5],
```

Comment:
I think we'll need to convert the `created_at` fields into strings or integers before serializing it to JSON.

Reply:
Trying `int(circle[5].timestamp())` but I don't know how to run the code so no idea if this works.